tekton/ci/jobs/tasks.yaml

```diff
@@ -129,4 +129,50 @@ spec:
         # Check failed. Return exit code 1.
         sys.exit(1)
       else:
-        print("Exactly one \"kind/*\" label found: {}".format(validKindLabels))
\ No newline at end of file
+        print("Exactly one \"kind/*\" label found: {}".format(validKindLabels))
+---
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: check-name-matches
+  namespace: tektonci
```

Comment:
@afrittoli I see there is an existing task in `tekton-ci` namespace, shall we make that as `ClusterTask` so that `Pipelines` in other namespace can reference the same task?

Reply:
To give you more context, the `tektonci` namespace contains the initial implementation of CI, which uses `PipelineResources` and `Conditions`.
The new version of CI runs in `tekton-ci` uses workspaces, results and `when` expressions.
If you want your job to use `when` expressions instead of conditions, it should then be migrated to the `tekton-ci` namespace and the `ci-workspace` folder.

Reply:
ahh...okay, thanks will move whole to `ci-workspace`